network-api/networkapi/wagtailpages/pagemodels/customblocks/recent_blog_entries.py

```diff
@@ -40,6 +40,8 @@ class RecentBlogEntries(blocks.StructBlock):
     #       error indication what's wrong.
 
     def get_context(self, value, parent_context=None):
+        from ..blog.blog_category import BlogPageCategory  # noqa
```

Comment:
This line should not be necessary here, given that it's already on line 7 as well.